### PR TITLE
Added dockerfile, docker-compose, updated readme

### PR DESCRIPTION
**Pull request by ashby-bowles**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM node:8.9.4
```
Comment (matthess91) [inline Dockerfile]: Any reason to not use a newer node ver. like `node:11-alpine`?  that'll be the current node and the slimmer base using alpine.

Comment (ashby-bowles) [reply Dockerfile]: fine with me. thats the node version WT website was using so wasn't sure if that was commonly used or something

Comment (matthess91) [reply Dockerfile]: oh interesting.  Their might be a good reason to use an older version but I'm not sure.

**Review by matthess91 — APPROVED** [review]
lgtm

**Pull request merged**